canonical/canonical.com
# chore: update secondary navigation and reorganize big data analytics

## Done

[Copy Doc](https://docs.google.com/document/d/1Y0dxbuar0UAPSCfILtP7fsEX-83yDh7Ha637FA9Wbik/edit?tab=t.0#heading=h.k4rz49xy6oy1)

- [x] rephrased "Edge-AI" to "Edge AI"
- [x] Added AI infrastructure to secondary navigation
- [x] Added big data analytics to secondary navigation  

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
    - Go to the solutions>ai page
    - Check if Edge-AI is rephrased to Edge AI
    - Check the two new links - AI infrastructure & Big data analytics

## Issue / Card

Fixes #[WD-16808](https://warthogs.atlassian.net/browse/WD-16808)

## Screenshots
<img width="1145" alt="image" src="https://github.com/user-attachments/assets/4b5bae82-1f67-4881-b2e3-11cb2ba23db9">
